### DISCUSSION TITLE
Sweden Discovery+ related channels changed logos

### DISCUSSION THREAD

**Discussion by AFrameWorker** [iptv-org/community#1016] [Q&A]
Hi, as of late, the logos have changed on the following channels and I'd like you to update them:

Kanal5.se [NEW Kanal 5 logo](https://images.ctfassets.net/989y85n5kcxs/5Daj4iCZBmFKCruRb8Xp0d/cb88f7b14d9281adefe6c73383f3be56/NEW_Kanal_5_HD__S_.png)
Kanal9.se [NEW Kanal 9 logo](https://images.ctfassets.net/989y85n5kcxs/1xr271GGqOAEQvbzYrUiww/d9ffa60842cfa7c23d3fda0d7af1bf3d/NEW_Kanal_9_HD__S_.png)
Kanal11.se [NEW Kanal 11 logo](https://images.ctfassets.net/989y85n5kcxs/2ism06ehlKvBxMZZExRX4v/2f7c1a6f5f900073d8b5d2ea4fb494e1/NEW_Kanal_11_HD__S_.png)

Sources/Sites: allente.se and tv.nu, I assume tv24.se will update theirs eventually too.

Unrelated to D+ channels, however still important:
I also did notice that the logo for BBC Nordic in Sweden (and in the rest of the Scandinavian countries too?) is incorrect, and still displays as "BBC Brit" - Probably due to the fact that the tvg-id is BBCBrit.se - this was what the channel was called in the past, but no longer is. I'd advice to change the tvg-id in allente.se/no/dk/fi to BBCNordic.uk instead, which is now the correct one.

(Or to simply add new tvg-id's for corresponding country)

**Comment by freearhey** [comment]
https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#how-to-edit-channel-description
